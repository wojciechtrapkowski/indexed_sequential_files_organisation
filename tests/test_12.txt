insert 1 100
insert 2 200
insert 3 300
insert 4 400
insert 5 500
insert 6 600
insert 7 700
insert 8 800
insert 9 900
insert 10 1000
insert 11 1100
insert 12 1200
insert 13 1300
insert 14 1400
insert 15 1500
insert 16 1600
insert 17 1700
insert 18 1800
insert 19 1900
insert 20 2000
print