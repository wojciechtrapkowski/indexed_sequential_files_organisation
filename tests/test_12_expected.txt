















Overflow area is full, reorganising




================================================
Index area
================================================
Page 0 number of entries: 4
	Entry 0
		start_key: 1
		page_index: 0
	Entry 1
		start_key: 5
		page_index: 1
	Entry 2
		start_key: 9
		page_index: 2
	Entry 3
		start_key: 13
		page_index: 3
================================================
Main area
================================================
Guardian overflow page index: null

Page 0 number of entries: 4
	Entry 0
		key: 1
		value: 100
		overflow_entry_index: null
	Entry 1
		key: 2
		value: 200
		overflow_entry_index: null
	Entry 2
		key: 3
		value: 300
		overflow_entry_index: null
	Entry 3
		key: 4
		value: 400
		overflow_entry_index: null
Page 1 number of entries: 4
	Entry 0
		key: 5
		value: 500
		overflow_entry_index: null
	Entry 1
		key: 6
		value: 600
		overflow_entry_index: null
	Entry 2
		key: 7
		value: 700
		overflow_entry_index: null
	Entry 3
		key: 8
		value: 800
		overflow_entry_index: null
Page 2 number of entries: 4
	Entry 0
		key: 9
		value: 900
		overflow_entry_index: null
	Entry 1
		key: 10
		value: 1000
		overflow_entry_index: null
	Entry 2
		key: 11
		value: 1100
		overflow_entry_index: null
	Entry 3
		key: 12
		value: 1200
		overflow_entry_index: null
Page 3 number of entries: 8
	Entry 0
		key: 13
		value: 1300
		overflow_entry_index: null
	Entry 1
		key: 14
		value: 1400
		overflow_entry_index: null
	Entry 2
		key: 15
		value: 1500
		overflow_entry_index: null
	Entry 3
		key: 16
		value: 1600
		overflow_entry_index: null
	Entry 4
		key: 17
		value: 1700
		overflow_entry_index: null
	Entry 5
		key: 18
		value: 1800
		overflow_entry_index: null
	Entry 6
		key: 19
		value: 1900
		overflow_entry_index: null
	Entry 7
		key: 20
		value: 2000
		overflow_entry_index: null
================================================
Overflow area
================================================
Page 0 number of entries: 0
Page 1 number of entries: 0
